README.md

```diff
@@ -1 +1,15 @@
-# zeroc-ice-py-ubuntu1404
\ No newline at end of file
+Ubuntu 14.04 Zeroc Ice Python Builder
+=====================================
+
+Builds Zeroc Ice wheel and RPM Python packages for Ubuntu 14.04.
+
+This can be used to create installable binary packages as an alternative to compiling from source using pip.
+
+    docker build -t builder .
+    docker run --rm -v $PWD/dist:/dist builder
+
+To build a different version of IcePy:
+
+    docker run --rm -v $PWD/dist:/dist builder /build.sh VERSION
```

Comment:
Minor, it we wanted to reduce the command to build non-default versions, we could define 3.6.4 as the default in `./build.sh`

```
ICE_VERSION=${1:-"3.6.4"}
```

and declare `/build.sh` as entrypoint rather than command

```
ENTRYPOINT ["/build.sh"]
```

so that `docker run --rm -v $PWD/dist:/dist builder x.y.z` would suffice. If this repo is meant to be temporary, probably not worth it.